lib/plugins/ray_trace.js

```diff
@@ -16,7 +16,11 @@ module.exports = (bot) => {
     if (block) return block
   }
 
-  bot.blockAtCursor = (maxDistance = 256) => {
+  bot.blockAtCursor = (maxDistance = 256, liquid = true) => {
+    if (typeof maxDistance === 'boolean') {
```

Comment:
Can we not do that ? It's confusing for users

Reply:
What do you mean?